site/.vitepress/config.ts

```diff
@@ -300,7 +280,7 @@ export default defineConfig({
                 items: [
                   {
                     text: "wrapSignatureWith6492",
-                    link: "/wrapSignatureWith6492",
+                    link: "/wrapWith6492",
```

Comment:
can we instead update the name of the document and make sure the names in that doc match up?